pom.xml

```diff
@@ -101,24 +102,24 @@
                 <!-- force dependency convergence. 2.8.4 is intended for jersey-media-json-jackson 2.25 -->
```

Comment:
change all these comments to `required for dependency convergence with transitive dependencies of jersey-media-jackson`